src/functions-and-arrays.js

```diff
@@ -1,41 +1,119 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(a, b) {
 
+  if (a > b) {
+    return a
+  }
+
+  else if (b > a) {
+    return b
+  }
+
+  else if (a == b) {
+    return a
+  }
+
+
+}
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(arr) {
+
+  if (arr.length != 0) {
+
+    let word = arr[0]
+
+    for (let i = 1; i < arr.length; i++) {
+      if (arr[i].length > word.length) {
+        word = arr[i]
+      }
+    }
+
+    return word;
+
 
+  }
+
+  else if (arr.length === []) {
+
+    result = null
+
```

Comment:
La longitud del array es un número por lo tanto está comprobación nunca cumplirá.

```suggestion
  else if (arr.length === 0) {

    result = null

```